docker-compose.yml

```diff
@@ -0,0 +1,80 @@
+version: '3'
+
+services:  
+  appwritelite:
+    container_name: appwritelite
```

Comment:
appwrite-lite would be more consistent to our naming conventions in this context